go/gregor/client/client.go

```diff
@@ -57,15 +57,23 @@ func (c *Client) Save(ctx context.Context) error {
 		return err
 	}
 
-	return c.Storage.Store(c.User, b)
+	localDismissals, err := c.Sm.LocalDismissals(ctx, c.User)
+	if err != nil {
+		return err
+	}
+	var ldm [][]byte
```

Comment:
Why not use the MsgID type?

Reply:
`MsgID` is an interface, it can only be serialized as bytes.